crypto/dh/dh_lib.c

```diff
@@ -31,7 +31,7 @@
  *    must display the following acknowledgement:
  *    "This product includes cryptographic software written by
  *     Eric Young (eay@cryptsoft.com)"
- *    The word 'cryptographic' can be left out if the rouines from the library
+ *    The word 'cryptographic' can be left out if the routines from the library
```

Comment:
this is silly and stupid but legal stuff often is.  do NOT change a single letter in the copyright.  i will not repeat this comment everywhere. :(